releasenotes/notes/remove-span-get-set-meta-c6fb2528d198414d.yaml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
What's the deal between these two similarly named files?

Reply:
one is for the deprecation on 0.x, but the exact same release note file was used/updated for the 1.x branch. we cannot do this, they need to exist as two different files 1 for the deprecation and 1 for the removal.